CONTRIBUTING.md

```diff
@@ -20,17 +20,21 @@ the official [Go language](https://github.com/golang/go/pulls).
 
 All new code is expected to be covered by tests.
 
-#### PR Checks
+#### Continuous Integration on Pull Requests
 
-We expect all PR checks to pass before we merge a PR, which can be investigated by following the `Details` links to CircleCI and CodeCov for unit/integration tests and code coverage checks.
+We expect all PR checks to pass before we merge a PR.
 
-![Screen Shot 2021-08-31 at 10 35 37 AM](https://user-images.githubusercontent.com/1819836/131533266-7c87305d-37df-4bd5-a9ea-6fb8e51e4b50.png)
-
-The code coverage report has a target of 90%. This is the goal, but is not a hard requirement. Reviewers ultimately make the decision about code coverage and quality and will merge PR's at their discretion. Any divergence from the expected 90% should be communicated by the reviewers to the PR author.
+The code coverage report has a target of 90%. This is the goal, but is not a hard requirement. Reviewers ultimately make the decision about code coverage and quality and will merge PRs at their discretion. Any divergence from the expected 90% should be communicated by the reviewers to the PR author.
 
 Please feel free to comment on a PR if there is any difficulty or confusion about any of the checks.
 
-#### What to expect
+Sometimes a pull request's checks will show failures that aren't related to its changes. When this happens, you can try the following steps:
+
+1. look through the gitlab job logs for an obvious cause
+2. retry the test a few times to see if it flakes
+3. ask the ddtrace-go chat channel for help
```

Comment:
```suggestion
3. for internal contributors, ask the #dd-trace-go channel for help
4. if you are not an internal contributor, [open an issue](https://github.com/DataDog/dd-trace-go/issues/new/choose) or ping @Datadog/apm-go
```

dd-trace-go gets quite a few of external contributors, so let's leave an option for them to reach the team as well!